backend/src/od.rs

```diff
@@ -31,22 +31,19 @@ impl DemandModel {
         }
     }
 
-    pub fn make_requests(&self, map: &MapModel) -> Vec<(RoadID, RoadID, usize)> {
+    pub fn make_requests(&self, map: &MapModel, fast_sample: bool) -> Vec<(RoadID, RoadID, usize)> {
         info!(
-            "Making requests from {} zones and {} desire lines",
+            "Making requests from {} zones and {} desire lines, sampling = {fast_sample}",
             self.zones.len(),
             self.desire_lines.len()
         );
 
-        // TODO Plumb through UI
-        // To speed up the impact calculation, how many specific requests per (zone1, zone2)? If
-        // true, just do one, but weight it by count.
-        let fast_sample = true;
-
         let mut rng = WyRand::new_seed(42);
         let mut requests = Vec::new();
 
         for (zone1, zone2, raw_count) in &self.desire_lines {
+            // To speed up the impact calculation, how many specific requests per (zone1, zone2)? If
+            // true, just do one, but weight it by count.
```

Comment:
I imagine theres a really big difference in both the performance and the accuracy when sampling 1 route vs sampling all the routes.

It might be worth exploring having the "fast_sample" do more than 1, still a small enough number that it feels relatively "instant". I don't think you should do that as part of this PR, just thinking about the future.

Reply:
Yep, agreed. Ideally I'd love to expose some kind of way for the user to say how long they're willing to wait, cut things off at that point, and give some sense of the accuracy achieved.